md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessTemplateAPI.java

```diff
@@ -454,31 +406,25 @@ RestWrapper apply(@ModelAttribute("processtemplate")
                     LOGGER.debug("processtempinfo.canrecover= " + processTempInfo.getCanRecover());
                     LOGGER.debug("processInfo next process id is " + processInfo.getNextProcessIds());
                     processTempInfo.setNextProcessIds(processInfo.getNextProcessIds());
-                    //s.selectOne("call_procedures.UpdateProcess", processTempInfo);
                 }
             }
             // Updating existing properties with changes made in properties template
             for (ProcessTemplate processTempInfo : processTemplateInfos) {
-                // List<Process> processInfos = s.selectList("call_procedures.select-process-list-for-template-id", processTempInfo);
                 List<Process> processInfos = processTemplateDAO.selectPListForTemplate(processTemplate.getProcessTemplateId());
                 for (Process processInfo : processInfos) {
-                    //   List<PropertiesTemplate> propTemplateList = s.selectList("call_procedures.ListPropertiesOfProcessTemplate", processTempInfo);
                     List<PropertiesTemplate> propTemplateList = propertiesTemplateDAO.listPropertiesTemplateBean(processTemplate.getProcessTemplateId());
                     if (propTemplateList.size() != 0) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use isEmpty() to check whether the collection is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1155)